auth-api/src/auth_api/resources/org.py

```diff
@@ -161,7 +162,7 @@ def put(org_id):
                                                           bearer_token).as_dict(), http_status.HTTP_200_OK
                 else:
                     response, status = org.update_org(org_info=request_json, token_info=toke_info,
-                                                      bearer_token=bearer_token).as_dict(), \
+                                                      bearer_token=bearer_token, origin_url=origin).as_dict(), \
```

Comment:
Is this part of this ticket ?

Reply:
For sending review account mail to the staff